extensions/coroutines-extensions/src/commonTest/kotlin/com/squareup/sqldelight/runtime/coroutines/runTest.kt

```diff
@@ -18,4 +18,9 @@ package com.squareup.sqldelight.runtime.coroutines
 
 import kotlinx.coroutines.CoroutineScope
 
-expect fun runTest(body: suspend CoroutineScope.() -> Unit)
+expect fun DbTest.runTest(body: suspend CoroutineScope.(TestDb) -> Unit)
+
+expect class AtomicInt(value_: Int) {
```

Comment:
just realized- you could add a testImplementation on deps.stately and get `AtomicInt` from that instead of implementing it manually. I can add that later if no one else gets around to it

Reply:
Ahh I didn't think to do that

Reply:
https://github.com/cashapp/sqldelight/pull/3673